vita3k/gui/src/app_selector.cpp

```diff
@@ -368,6 +369,16 @@ void draw_app_selector(GuiState &gui, HostState &host) {
         const auto SIZE_APP_LIST = ImVec2((host.cfg.apps_list_grid ? 840.f : 900.f) * scal.x, display_size.y - POS_APP_LIST.y);
         ImGui::SetNextWindowPos(host.cfg.apps_list_grid ? POS_APP_LIST : ImVec2(1.f, POS_APP_LIST.y), ImGuiCond_Always);
         ImGui::BeginChild("##apps_list", SIZE_APP_LIST, false, ImGuiWindowFlags_NoBackground | ImGuiWindowFlags_NoTitleBar | ImGuiWindowFlags_NoCollapse | ImGuiWindowFlags_NoSavedSettings);
+
+        // Set Scroll Pos
+        current_scroll_pos = ImGui::GetScrollY();
+        max_scroll_pos = ImGui::GetScrollMaxY();
+        if (scroll_type != 0) {
+            const float scroll_move = (scroll_type == -1 ? 340.f : -340.f) * scal.y;
+            ImGui::SetScrollY(ImGui::GetScrollY() + scroll_move);
+            scroll_type = 0;
+        }
```

Comment:
```suggestion
        ImGui::SetScrollY(ImGui::GetScrollY() + scroll_type * 340.f * scal.y);
        scroll_type = 0;
```
? Looks like a scalar that would make this possible.

Reply:
i no understund your code here scroll-type is just for said if is + or - so how 1 x 340 can done same

Reply:
I have a history of making stupid suggestions but how I thought this one through:
 - `scroll_type == 1`? `scroll_type * 340.0f == -340.0f`
 - `scroll_type == 0`? `scroll_type * 340.0f == 0.0f`
 - `scroll_type == -1`? `scroll_type * 340.0f == 340.0f`

I thought that matches the ternary a bit better `(scroll_type == -1 ? 340.f : -340.f)`.

Reply:
i just understund here, you said i can remove first if, for other you sugest is exactly same i have set

Reply:
i keep it, change code just don't works